src/Caching/samples/Directory.Build.props

```diff
@@ -1,8 +1,8 @@
 <Project>
-
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
 
   <PropertyGroup>
-    <PackageId>Microsoft.AspNetCore.BenchmarkRunner.Sources</PackageId>
+    <IsPackable>false</IsPackable>
```

Comment:
I think most of the places where you have added IsPackable should already be covered by the conventions expressed in the repo root Directory.Build.props. Are these broken by Arcade?

E.g. samples, perf projects, and test projects should already be set to false

Reply:
It seems like they are broken by Arcade. It's not clear to me in what order files are imported in Arcade, but I couldn't seem to make generalized IsPackable rules work. So I figured I' do the thing that works for now and filed https://github.com/aspnet/Extensions/issues/915 to re-examine this later.